src/gda/bnxt/backend_gda_bnxt.cpp

```diff
@@ -216,6 +216,11 @@ void GDABackend::create_qps(int sq_length) {
     /* Alloc QP */
     qps[i] = bnxtdv_ftable_.create_qp(pd_orig, &bnxt_qps[i].attr);
     CHECK_NNULL(qps[i], "bnxt_re_dv_create_qp");
+
+    /* Set hardware port idx.
+     * This is abstracted from the user but the hardware alternates between port 0 and 1
+     */
+    bnxt_qps[i].port = (i % 2);
```

Comment:
can we use ibv_query to know how many ports? It may be hidden for bonded ifaces?